.detekt.yml

```diff
@@ -0,0 +1,4 @@
+style:
+  # We want to allow "TODO" in comments
```

Comment:
This is ok for now, but depending on how we want to approach the integration we might need to settle with whatever [a-c does](https://github.com/mozilla-mobile/android-components/blob/5c56606626d542d282ec2619e75b0d5e2cd64486/config/detekt.yml), otherwise we'll break klint there. Not for this PR, just for future reference.